libs/blog/articles/ui-article-card/src/lib/components/article-regular-card/article-regular-card.component.html

```diff
@@ -1,25 +1,31 @@
 <div
-  class="rounded-lg shadow-none hover:shadow-[0_0_0_1px_rgba(231,4,100,1)]"
+  class="group relative flex h-full w-full rounded-lg shadow-none transition-transform hover:scale-105 hover:shadow-[0_0_0_1px_rgba(231,4,100,1)]"
```

Comment:
Hmm isn't it a similar problem for users who have disabled motions? tailwind exposes utils to ignore motions if user prefers them to be reduced.
 https://tailwindcss.com/docs/transition-property#prefers-reduced-motion